docs/en/docs/release-notes.md

```diff
@@ -5,6 +5,17 @@ hide:
 
 # Release Notes
 
+## Unreleased
+
+### Added
+
+- Allow passing HTTP/WebSocket handlers directly to routes. They are automatically wrapped in Gateways-
+
+### Changed
+
+- Pluggables can now receive plain Extensions and Extension classes.
+- Rename to extensions.
```

Comment:
- **Breaking**: Pluggables are now renamed to `Extensions`, check the docs for more information.